From b586bbeb393d2cc78e7cf560302c4c6391e866ae Mon Sep 17 00:00:00 2001
From: Ghislain Antony Vaillant <ghisvail@users.noreply.github.com>
Date: Thu, 2 Mar 2017 12:34:56 +0000
Subject: [PATCH 2/4] Fixup project version

The current release is tagged `v0.6.2` but the project version was still at `0.6.1`
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ee01e7c..d806cd8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.1)
-project(docopt.cpp VERSION 0.6.1)
+project(docopt.cpp VERSION 0.6.2)
 
 include(GNUInstallDirs)
 
-- 
2.21.0 (Apple Git-122.2)

